autogpt/utils.py

```diff
@@ -49,15 +49,17 @@ def readable_file_size(size, decimal_places=2):
     return f"{size:.{decimal_places}f} {unit}"
 
 
-def get_bulletin_from_web() -> str:
```

Comment:
why remove the type hint here?

Reply:
It’s from a diff pr, this one needs help rebasing

Reply:
yeah, it's in a very bad state atm though that rebasing won't help. There are broken things added like this https://github.com/Significant-Gravitas/Auto-GPT/commit/f4db77a7b0513effbc317d5b374fdd9ad9ecf161 where the merge conflict was committed and then it references code that doesn't exist in either upstream or that branch...